doc/source/whatsnew/v0.10.0.rst

```diff
@@ -184,12 +184,10 @@ labeled the aggregated group with the end of the interval: the next day).
 
     import io
 
-    data = ('a,b,c\n'
-            '1,Yes,2\n'
-            '3,No,4')
+    data = "a,b,c\n1,Yes,2\n3,No,4"
```

Comment:
@jorisvandenbossche made the valid point that this is slightly harder to understand as an input to read_csv, so maybe we use a multiline string similar to this instead (which black is fine with):
```python
data = """
a,b,c
1,Yes,2
3,No,4
"""
```

What do you think @Meghana-12 @MarcoGorelli ?

Reply:
I'll be making changes in a min then.

Reply:
This same pattern appears in several places within doc/source/user_guide/io.rst, so if you wanted to tackle those in another PR that would also be very welcome

Reply:
Sure, I'll have a look at it!